.github/workflows/nodejs.yml

```diff
@@ -29,6 +29,6 @@ jobs:
     - run: yarn autocomplete
     - run: yarn format:ci
     - run: yarn tslint
-    - run: yarn test --testPathIgnorePatterns='.*benchmark.*'
```

Comment:
This is not necessary, `yarn test` runs exactly the script you change in `package.json`.

Reply:
not sure what you mean. before, the flags were being added here. i've moved the flags to be added in package.json. isn't this the same as what you're suggesting?

Reply:
Oops sorry. Misread the diff..